docs/src/index.md

```diff
@@ -1,12 +1,55 @@
-# PowerSystems.jl
+# PRASInterface.jl
 
 ```@meta
 CurrentModule = PRASInterface
 ```
 
-## Overview
+## About
+
+`PRASInterface.jl` is a [`Julia`](http://www.julialang.org) package that provides an interface to [`PRAS.jl`](https://nrel.github.io/PRAS) from [Sienna](https://www.nrel.gov/analysis/sienna.html)'s [`PowerSystem.jl`](https://github.com/NREL-Sienna/PowerSystems.jl)'s `System` data model.
+
+The Probabilistic Resource Adequacy Suite (PRAS) analyzes the resource adequacy of a bulk power system using monte carlo methods.
```

Comment:
Couly you please chsne this to Monte Carlo?

Reply:
Done.